data_juicer/ops/base_op.py

```diff
@@ -20,13 +20,37 @@ def __init__(self, text_key: str = None):
 
     def process(self, sample):
         """
-        For sample level, sample --> sample or sample --> [samples, ...]
+        For sample level, sample --> sample
 
         :param sample: sample to process
         :return: processed sample
         """
         raise NotImplementedError
 
+class BatchMapper(Mapper):
```

Comment:
There is no substantial difference between `Mapper` and `BatchMapper` classes in terms of signature and implementation. In other words, there is no need for us to introduce `BatchMapper`?

Reply:
That's right.